pages/desktop/user.py

```diff
@@ -119,8 +119,15 @@ def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
 
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._email_checkbox)
+        if not email_checkbox.is_selected():
```

Comment:
I added the wait above which verifies that the checkbox displays, this particular "if" determines whether or not the checkbox is already selected. I don't understand why another wait would be appropriate as the status of whether or not the checkbox is selected doesn't change with time. It does make sense to add a wait after line 125 as the state should then match the expected state.